mutablesecurity/solutions/implementations/duplicati/meta.yaml

```diff
@@ -0,0 +1,8 @@
+full_name: Duplicati 2.0
+description: Duplicati is a backup client that securely stores encrypted, incremental, compressed backups on local storage, cloud storage services and remote file servers. 
+references:
+  - https://www.duplicati.com/
+  - https://github.com/duplicati/duplicati
+maturity: DEV_ONLY
```

Comment:
After solving all the change requests, let's set the maturity to `PRODUCTION`. We will push this whole module into a future version of MutableSecurity 🎉.

Reply:
done